### PR TITLE
Submit data to data store

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
Not ready for review yet

File: actions/actions.py [hunk 1]
```diff
@@ -601,6 +601,11 @@ async def submit(
                 "User-Agent": "rasa/covid19-healthcheckbot",
             }
 
+            # For dev purposes only, set the env for MSISDN since rasa shell uses uuid
+            # which fails validation against the eventstore
+            if config.DEV_MSISDN:
+                post_data.update({"msisdn": config.DEV_MSISDN})
+
```
Comment (miltontony) [inline actions/actions.py:608]: Wasn't sure of a nice way to do this, but this is handy to test submission to the api.

**Comment by rudigiesler** [comment]
> @rudigiesler I had added the lat/long fields to the mappings because of some weird behaviour. If you don't add them, they are only set during the first form run. When the user starts again and the slots are pre-filled, the lat/long fields are blank.

Ah right, yes we want the fields on the list of fields that we copy over. I think it's better if we add it to that action directly.

File: actions/actions.py (hunk 1 above)
Comment (rudigiesler) [reply actions/actions.py:610]: You can do a `rasa shell --conversation-id 27820001001`, so I'll remove this

Comment (miltontony) [reply actions/actions.py]: 🎉

**Comment by miltontony** [comment]
👍 awesome!

**Pull request merged**